src/app/children/child-details/child-details.component.ts

```diff
@@ -133,16 +138,19 @@ export class ChildDetailsComponent implements OnInit {
           this.initForm();
           this.entityMapperService.load<School>(School, this.child.schoolId)
             .then(school => this.currentSchool = school);
+          this.blobService.getImage(this.child.photoFile)
```

Comment:
maybe we could extract the image loading into a component or directive, so that this doesn't have to be repeated in child-details, child-block and other places.